### PR TITLE
Update JoomlaWebdriverTestCase.php

### PR DESCRIPTION
**Pull request by kshitijSharma2014**
added a test to check linked items on the site.

### CODE REVIEW

**Comment by kshitijSharma2014** [comment]
@puneet0191  @javigomez  @gunjanpatel  please have a look

File: tests/system/webdriver/tests/JoomlaWebdriverTestCase.php [hunk 1]
```diff
@@ -197,4 +197,13 @@ public function helpScreenshot($fileName, $folder = null)
 			return $fileName;
 		}
 	}
+
```
Comment (javigomez) [inline tests/system/webdriver/tests/JoomlaWebdriverTestCase.php:200]: Missing method docblock. See my previous comment in the other pull and http://joomla.github.io/coding-standards/?coding-standards/chapters/docblocks.md

File: tests/system/webdriver/tests/JoomlaWebdriverTestCase.php [hunk 2]
```diff
@@ -197,4 +197,13 @@ public function helpScreenshot($fileName, $folder = null)
 			return $fileName;
 		}
 	}
+	
+	public function itemExist($ItemName)
+	{
+		$d = $this->driver;
```
Comment (javigomez) [inline tests/system/webdriver/tests/JoomlaWebdriverTestCase.php]: All variables names need to be selfdescriptive. I know this $d is already in this class everywhere, but for me we should rename it $driver or something that descrives clearly the what this variable contains. A variable name of less than 3 chars is usually not good name. I fact is one of the things that you can scan for with the PHP Mess detector (http://phpmd.org/rules/naming.html#shortvariable). You don't need to change it now, but lets consider it for the future pulls.

File: tests/system/webdriver/tests/JoomlaWebdriverTestCase.php [hunk 3]
```diff
@@ -197,4 +197,13 @@ public function helpScreenshot($fileName, $folder = null)
...
+		$element = $d->findElement(By::xPath("//a[contains(text(),'" .  $ItemName . "')]"));
+		if($element!=null)
+			return true;
+		return false;
```
Comment (javigomez) [inline tests/system/webdriver/tests/JoomlaWebdriverTestCase.php:214]: All control structures must contain their logic within braces.

Please check http://joomla.github.io/coding-standards/?coding-standards/chapters/php.md in it's section "Control Structures".

The code should look like:

``` php
if($element!=null)
{
    return true;
}

return false;
```

**Comment by kshitijSharma2014** [comment]
@javigomez  I have edited this pull. please review it
thanks :)

File: tests/system/webdriver/tests/JoomlaWebdriverTestCase.php [hunk 4]
```diff
@@ -197,4 +197,20 @@ public function helpScreenshot($fileName, $folder = null)
 			return $fileName;
 		}
 	}
+	
+	/**
+	 * check whether the item is present on the site or not given the xpath variable.
+	 *  @param string $itemName   stores the name of the item to be searched
+	 *  @param string $arg	xpath variable
+	 */
```
Comment (puneet0191) [inline tests/system/webdriver/tests/JoomlaWebdriverTestCase.php:205]: Where is return? in the comment?

Comment (puneet0191) [inline tests/system/webdriver/tests/JoomlaWebdriverTestCase.php:205]: phpcs doesn't work for you I think

**Comment by puneet0191** [comment]
Why do you want to add this Function in this class?, are we going to use it in most of the tests class? Why are you passing Xpath as a parameter? I don't think this is needed

**Comment by kshitijSharma2014** [comment]
@puneet0191  I am using this function in my tests to locate the item on the front end, also the xpath I am passing is just a letter like 
'a' to check whether it is a link 
'h3' to check if it is of h3 type or not, according to what I am getting through its xpath.

I am not putting whole of xpath in the function parameter , just a letter.

**Comment by puneet0191** [comment]
Not merging this one!

**Pull request closed without merging**